crawl-ref/source/dat/strings/de/adjectives-e.txt

```diff
@@ -570,7 +570,7 @@
 ################################
 %%%%
 "vorpal "
-"Vorpal-"
+"scharfe "
```

Comment:
Also not sure about this one. Vorpal is a specific fantasy thing. It doesn't just mean sharp.

Reply:
I know, but vorpal isn't well-known in German, and it's hard to translate. Sharp or deadly come closest. The dnddeutsch translator goes with 'Vorpal sword' -> 'Henkersschwert'. Which isn't bad, but sounds more like a specific type of sword, not a DCSS weapon brand. Christian Enzensberger uses 'scharfgebifftes Schwert' and 'biffe Klinge' in his translation of the Jabberwocky (Der Zipferlake), which seems to be the most popular translation. Perhaps we should use 'scharfgebiffte'?

Reply:
What is biffe/gebifftes. Is it just a made up word like vorpal?